app/scripts/controllers/app-state.js

```diff
@@ -10,6 +11,8 @@ import {
   POLLING_TOKEN_ENVIRONMENT_TYPES,
 } from '../../../shared/constants/app';
 
+const APPROVAL_REQUEST_TYPE = 'unlock';
```

Comment:
Harmless currently, but just for reference, I've created a future ticket to address these types.

Potentially to create an `APPROVAL_TYPE` enum, probably in core, so these can be centrally defined meaning they can be checked and created from different places, plus synchronised across the different clients.